apps/monarch/monarch-frontend/vite.config.ts

```diff
@@ -27,9 +27,9 @@ export default defineConfig({
     }),
   ],
 
-  // define: {
-  //   global: {}
-  // },
+  define: {
```

Comment:
Setting `global: 'globalThis` allows the app to run locally, as well as pass tests that would normally fail if it was set to `{}` as well